src/test/java/uk/nhs/tis/trainee/notifications/service/ProgrammeMembershipServiceTest.java

```diff
@@ -283,7 +297,13 @@ void shouldAddInAppNotificationsWhenNotExcludedAndMeetsCriteria(NotificationType
     Map<String, Object> variables = variablesCaptor.getValue();
     assertThat("Unexpected programme name.", variables.get(PROGRAMME_NAME_FIELD),
         is(PROGRAMME_NAME));
+    assertThat("Unexpected programme number.", variables.get(PROGRAMME_NUMBER_FIELD),
+        is(PROGRAMME_NUMBER));
     assertThat("Unexpected start date.", variables.get(START_DATE_FIELD), is(START_DATE));
+    if (notificationType.equals(LTFT) || notificationType.equals(DEFERRAL)
+        || notificationType.equals(SPONSORSHIP)) {
```

Comment:
Is there a reason not to check E_PORTFOLIO or INDEMNITY_INSURANCE here?

Reply:
As the E_PORTFOLIO and INDEMNITY_INSURANCE do not have any mailto link, so the gmcNumber is not passed in there in the [createInAppNotifications()](https://github.com/Health-Education-England/tis-trainee-notifications/blob/7d8340bfbd159f4b8674c9c225e2e426c16c5b6c/src/main/java/uk/nhs/tis/trainee/notifications/service/ProgrammeMembershipService.java#L265)